zebrad/tests/common/configs/v1.0.0-rc.4.toml

```diff
@@ -0,0 +1,69 @@
+# Default configuration for zebrad.
```

Comment:
Copied from the test error message:
https://github.com/ZcashFoundation/zebra/actions/runs/3998037053/jobs/6860159405#step:14:3835